test/support/error_builder.ex

```diff
@@ -0,0 +1,10 @@
+defmodule DeliriumTremex.ErrorBuilder do
```

Comment:
What is this and where is it used?

Reply:
It's needed for the tests to pass because of the changes in 93c5eee52204849905d63f8031290061d19dc26f.

Reply:
OK